apps/utils/scripts/build-bfd

```diff
@@ -37,7 +37,7 @@ of maven arguments.
 -a          Build all. Same as -ijs.  Use before commit!
 -t          Enable unit tests.
 -i          Enable unit and IT tests.
--j          Enable javadoc generation.
+-j          Enable swagger and javadoc generation.
```

Comment:
Open to feedback on this. We _could_ make this a separate flag, but this script already has a ton of flags, and this didn't seem worthy of adding another.